.gitignore

```diff
@@ -30,11 +30,11 @@ fix/chem
 fix/cice
 fix/cpl
 fix/datm
-fix/gdas
 fix/gldas
 fix/gsi
 fix/lut
 fix/mom6
+fix/mon
```

Comment:
Can we make this "monitor", maybe even "da_monitor"? I want to get in the habit of using descriptive names instead of short inscrutable ones.

Reply:
Also see below about whether this should be `fix` at all instead of `parm`.

Reply:
Yeah...I am moving these "fix" files to `parm/monitor` (which is being renamed from `parm/mon`. Will discontinue use of `fix/mon`.